src/rp2_common/hardware_sync/include/hardware/sync.h

```diff
@@ -70,6 +70,11 @@ typedef volatile uint32_t spin_lock_t;
 #define PICO_SPINLOCK_ID_HARDWARE_CLAIM 11
 #endif
 
+// PICO_CONFIG: PICO_SPINLOCK_ID_RAND, Spinlock ID for Random Number Generator, min=0, max=31, default=11, group=hardware_sync
```

Comment:
Huh? Says "default=11" here, but then "PICO_SPINLOCK_ID_RAND 12" a couple of lines further down?

Reply:
Thanks for the spot.  Now fixed.